.github/workflows/tests.yaml

```diff
@@ -49,7 +49,12 @@ jobs:
 
     strategy:
       matrix:
-        repo: ["Workiva/over_react", "rrousselGit/provider", "dart-lang/args"]
+        repo: [
+          "Workiva/over_react",
+          "Workiva/w_module",
```

Comment:
w_module has a nested `pubspec.yaml` file in [example](https://github.com/Workiva/w_module/blob/master/example/pubspec.yaml)

Add it as a consumer to verify this new functionality of not breaking on nested packages